src/main/java/org/mskcc/cbio/oncokb/service/UserService.java

```diff
@@ -438,6 +442,10 @@ public Optional<UserDTO> updateUser(UserDTO userDTO) {
         return updatedUserDTO;
     }
 
+    public Optional<UserDTO> updateUserBeforeTrialAccountActivation(UserDTO userDTO) {
```

Comment:
What's the purpose of adding this wrapper?

Reply:
I added a helper function `updateUserFromUserDTO` which handles the logic of updating the user from the given UserDTO (this was already in `updateUser`). My new function `updateUserBeforeTrialAccountActivation` needs this same logic, but it shouldn't expire the trial account when updating like `updateUser` does. I added this wrapper because I think the name makes the most sense for the use case, and `updateUserFromUserDTO` would not be a descriptive name to expose as an API.